### PR TITLE
Tpc add trending and checkers

### PR DESCRIPTION
**Pull request by rmunzer**

### CODE REVIEW

File: scripts/generate-all-dpl-workflows.sh [hunk 1]
```diff
@@ -114,8 +114,8 @@ check_pwd
 ./tpc-full-calib-trend-qcmn.sh
 ./tpc-full-qcmn.sh
 ./tpc-full-nodummy-qcmn.sh
-./tpc-full-nodummy-noraw-qcmn.sh
-./tpc-full-nodummy-nopid-qcmn.sh
```
Comment (knopers8) [inline scripts/generate-all-dpl-workflows.sh:118]: if you are removing these two from here, please consider removing the scripts themselves.

File: scripts/generate-all-dpl-workflows.sh [hunk 2]
```diff
@@ -114,8 +114,8 @@ check_pwd
...
+./tpc-full-nodummy-qcmn-pp.sh
+./tpc-full-nodummy-qcmn-pp-v2.sh
 ./tpc-full-nodummy-qcmn-pp.sh
```
Comment (knopers8) [inline scripts/generate-all-dpl-workflows.sh:119]: `./tpc-full-nodummy-qcmn-pp.sh` is there twice

File: scripts/generate-all-dpl-workflows.sh (hunk 1 above)
Comment (rmunzer) [reply scripts/generate-all-dpl-workflows.sh:118]: Thanks. Done

File: scripts/generate-all-dpl-workflows.sh (hunk 2 above)
Comment (rmunzer) [reply scripts/generate-all-dpl-workflows.sh]: Thanks. Done

**Review by knopers8 — APPROVED** [review]

**Pull request merged**